adal/src/Microsoft.IdentityModel.Clients.ActiveDirectory/UserCredential.cs

```diff
@@ -29,54 +29,35 @@
 
 namespace Microsoft.IdentityModel.Clients.ActiveDirectory
 {
-    internal enum UserAuthType
-    {
-        IntegratedAuth,
-        UsernamePassword
-    }
+    using UserAuthType = Microsoft.Identity.Core.UserAuthType;
+
 
     /// <summary>
     /// Credential used for integrated authentication on domain-joined machines.
```

Comment:
This seems like a very thin wrapper, would it be possible to call the base class in core directly instead?

Reply:
I believe we would have to keep a public `UserCredential` in ADAL to remain backward compatibility? We could, however, put that base class in Core yet still gives it the same namespace as ADAL, but it would look somewhat weird to me. Please let me know if there is a better way to do it.